db/migrations/postgres/000092_add_pin_namespace.up.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+ALTER TABLE pins ADD COLUMN namespace VARCHAR(64);
+UPDATE pins SET namespace = 'ff_system';
```

Comment:
I've added a `migration_consideration` tag, as we've chosen `ff_system` really only because it's a reserved namespace name that can be used to query these values back (definitely *not* because these pre-upgrade pins were all for the `ff_system` namespace).

So we should document that.

Reply:
Yea, originally I was going to leave it blank but then realized the API would give you no way to query them (and that seemed bad).

Pins were treated as "global" in 1.0, and the aggregator loop that processes pins even logs them as if they are part of `ff_system` (see [here](https://github.com/hyperledger/firefly/blob/30b0c569639a8b74c5b18144f5779ff39190ac16/internal/events/aggregator.go#L93)), so it didn't feel terribly wrong to pick `ff_system` for this purpose. But it was always a bit inaccurate, as they are specific to a namespace even if they weren't treated that way.

Anyway, bit of a ramble but I agree we need to document it.